docs/src/main/paradox/.htaccess

```diff
@@ -10,3 +10,6 @@ RedirectMatch 301 ^(.*)/formFieldList.html$ $1/formFieldSeq.html
 RedirectMatch 301 ^(.*)/extractHostName.html$ $1/extractHost.html
 
 RedirectMatch 301 ^(.*)/release-notes.html$ $1/release-notes/
+
+RedirectMatch 301 ^(.*)/scala/http/(.*)$ $1/$2?language=scala
```

Comment:
Ooh, really nice, if that works ;)

Reply:
looks like: https://doc.akka.io/docs/akka-http/10.0.11+154-91b7071b+20171222-1715/scala/http/routing-dsl/index.html

Reply:
Very cool, just trying is always the best solution. :+1: